mount.go

```diff
@@ -32,6 +33,30 @@ import (
 	"github.com/jacobsa/timeutil"
 )
 
+func isMetadataPrefetchOnMountEnabled(metadataPrefetchMode string) bool {
+	switch metadataPrefetchMode {
+	case config.MetadataPrefetchOnMountSynchronous, config.MetadataPrefetchOnMountAsynchronous:
+		return true
+	}
+	return false
+}
+
+func isTypeCacheEnabled(metadataCacheTTL time.Duration, typeCacheMaxSizeMB int) bool {
+	if metadataCacheTTL <= 0 || typeCacheMaxSizeMB == 0 || typeCacheMaxSizeMB < -1 {
```

Comment:
check if `typeCacheMaxSizeMB < -1` is required or not. if not why and if yes why.